scales-position.Rmd

```diff
@@ -38,6 +95,59 @@ For more information on scale transformations see Section \@ref(scale-transforma
 ### Limits
 
 
+
+### Transformation {#scale-transformation}
```

Comment:
I put this in scales-guides, so you should probably remove from there — I think it makes sense to include here since you usually use with position scales, and then in the theory chapter we'd show how they also apply to other scales